.github/workflows/build.yml

```diff
@@ -44,6 +44,7 @@ jobs:
             libharfbuzz-dev liblzma-dev libunwind-dev libunwind-dev libvulkan1 \
             libx11-dev libxcb-render0-dev libxcb-shape0-dev libxcb-xfixes0-dev \
             libxmu-dev libxmu6 libegl1-mesa-dev llvm-dev m4 xorg-dev sway \
+            libgtk-3-dev libxdo-dev \
```

Comment:
If muda isn't working on Linux, let's just add muda in macOS and Windows dependencies only.